scripts/template/Component.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Box, system } from '@marigold/system';
+
+type Props = {};
```

Comment:
Maybe here COMPProps?

Reply:
need to insert an extra sed for it but we can do it :+1:

Reply:
I think it's better that we have separate names like in the other comps :)